internal/app/go-filecoin/node/block_propagate_test.go

```diff
@@ -42,7 +41,7 @@ func requireMineOnce(ctx context.Context, t *testing.T, minerNode *Node) *block.
 	worker, err := minerNode.CreateMiningWorker(ctx)
 	require.NoError(t, err)
 
-	// Miner should win first election as it has all the power so only
+	// Provider should win first election as it has all the power so only
```

Comment:
nit: change back to Miner which makes more sense in this context.  It's making blocks not deals.